backend/db/users.py

```diff
@@ -0,0 +1,7 @@
+from sqlmodel import SQLModel, Field
+
+class Users(SQLModel, table=True):
+    __tablename__ = "users"
+    user_id: int = Field(default = None, primary_key = True)
+    user_email: str = Field(nullable=False)                      # nullable=False - shouldn't be null
```

Comment:
```suggestion
    user_email: str = Field(nullable=False)                  
```
Redundant comment :)